index.html

```diff
@@ -240,9 +240,9 @@ <h1 class="contact-title">Contact Me</h1>
                      that needs coding.I’d love to help with it 
                 </p>
                 <form id="form_id" action="https://formspree.io/f/mlekbobg" method="post" class="contact">
-                        <input type="text" id="name" name="name" placeholder="enter your name" required maxlength="30">
-                        <input type="email" id="email" name="email" placeholder="enter your email" required>
-                        <textarea name="message" id="" cols="30" rows="10" required placeholder="Enter the message" maxlength="500"></textarea>
+                        <label for="name"><input type="text" id="name" name="name" placeholder="enter your name" required maxlength="30"></label>
+                        <label for="email"><input type="email" id="email" name="email" placeholder="enter your email" required></label>
+                        <textarea for="textarea" name="message" id="" cols="30" rows="10" required placeholder="Enter the message" maxlength="500"></textarea>
                     <button type="submit" name="submit">Get in touch</button>
```

Comment:
This button doesn't have the same colors as in figma template and this causes contrast issues 
Please refer to figma and make these changes also for hover effect 

![get in touch contrast](https://user-images.githubusercontent.com/59707859/221915452-9b65f4b7-9a6f-4064-8228-cc8d1611dd76.PNG)

Reply:
Thank you for notifying the correction. issue been resolved